ui/00_message/pawn/p0002.xml

```diff
@@ -2,51 +2,70 @@
   <message>
     <original>あ！　か、覚者さま！
 こんにちは――</original>
+	<translation>Ah! Hello, Master!</translation>
   </message>
   <message>
     <original>覚者――さま――？</original>
+	<translation>Master?</translation>
   </message>
   <message>
     <original>こんな自分でよければ、お手伝いを――！</original>
+	<translation>If you do not mind, Master! I may be of help.</translation>
   </message>
   <message>
     <original>あ、あの――覚者さま！
 自分もぜひ、旅のお伴を――！</original>
+	<translation>Oh, Master! I'd be honored to travel with you!</translation>
   </message>
   <message>
     <original>あの、覚者さま！
 役に立てることがあれば、自分を――！</original>
+	<translation>Ah, Master! 
+I am here to help in any way I can!</translation>
   </message>
   <message>
     <original>出番は――まだ――？</original>
+	<translation>Is it my turn yet?</translation>
   </message>
   <message>
     <original>少しでも、お手伝いできれば――</original>
+	<translation>I can help, even if its little.</translation>
   </message>
   <message>
     <original>自分にも何かできるはず――</original>
+	<translation>I can do something for myself.</translation>
   </message>
   <message>
     <original>ま、まだできてないんです――</original>
+	<translation>Well, I have not done it yet.</translation>
```

Comment:
Really small thing, here. Usually, I think you're right translating "ま" as "well", but I believe this time it's a stammer, rather than a meaningful word. 
"I, I still can't do it..."

Reply:
Fixed, will be it next updated commit